kernel/config.txt

```diff
@@ -21,12 +21,113 @@ necessary. Information about pre-built hardware as a whole will not be covered.
 [0.0] Index
 ________________________________________________________________________________
 
-- Busybox compatibility                                                    [1.0]
-- Never lose your .config ever again                                       [2.0]
-- Removing the perl requirement                                            [3.0]
+- Getting a config                                                         [1.0]
+- Modifying a config                                                       [2.0]
+- Busybox compatibility                                                    [3.0]
+- Never lose your .config ever again                                       [4.0]
+- Removing the perl requirement                                            [5.0]
 
 
-[1.0] Busybox compatibility
+[1.0] Getting a config
+________________________________________________________________________________
+
+The first step in this process is creating a .config file in the Linux source
+tree. It is essentially just a text file containing KEY=value pairs (along with
+comments), each of which control something in the linux build process. Some of
+them are features to be included, some values to be hardcoded.
```

Comment:
I think this can be omitted:

`Some of them are features to be included, some values to be hardcoded.`